### PR TITLE
Style/applications

### PR DESCRIPTION
**Pull request by riqthedev**
## Description

Fixed Mentor/Mentee application and signup styling to correct color scheme. Added Dropdown feature to Mentor application to match wireframe.

##Loom:
https://www.loom.com/share/6f9a162a0dc749da876cf81cdc1d386b

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.modalStyle .ant-modal-title /*For Edit Profile Button*/{
+    color: white;
+}
+
+.cEQGvI /*Navbar background color*/ {
+    background-color:#003D71;
+    
+
```
Comment (cainpalmer) [inline src/styles/styles.css:17]: clean up spacing?

**Review by cainpalmer — CHANGES_REQUESTED** [review]
PR is good length and Loom video clear, however besides the spacing that can be cleaned up it seems there are some conflicting files.

**Review by tinaxgao — CHANGES_REQUESTED** [review]
Thanks for the update, the styling changes are clear. my team will continue working on the signup forms after your update. Just take a look at the current conflicts.

**Review by tinaxgao — APPROVED** [review]

**Review by cainpalmer — APPROVED** [review]
Nice changes! looks great!

**Pull request merged**